test.sh

```diff
@@ -30,109 +30,119 @@ function setup_osbs() {
   OSBS_CLIENT_REPO=${OSBS_CLIENT_REPO:-https://github.com/containerbuildsystem/osbs-client}
   OSBS_CLIENT_BRANCH=${OSBS_CLIENT_BRANCH:-master}
 
-  # Pull fedora images from registry.fedoraproject.org
-  if [[ $OS == "fedora" ]]; then
-    IMAGE="registry.fedoraproject.org/$IMAGE"
-  fi
-
   # PIP_PREFIX: osbs-client provides input templates that must be copied into /usr/share/...
   ENVS='-e PIP_PREFIX=/usr'
   RUN="$ENGINE exec -i ${ENVS} $CONTAINER_NAME"
-  PYTHON="python$PYTHON_VERSION"
-  if [[ $OS == "fedora" ]]; then
-    PIP_PKG="python$PYTHON_VERSION-pip"
-    PIP="pip$PYTHON_VERSION"
-    PKG="dnf"
-    ENABLE_REPO="--enablerepo=updates-testing"
-    PKG_EXTRA="dnf-plugins-core desktop-file-utils flatpak ostree skopeo python$PYTHON_VERSION-libmodulemd glibc-langpack-en"
-    BUILDDEP="dnf builddep"
-    PYTHON="python$PYTHON_VERSION"
-  else
-    PIP_PKG="python-pip"
+
+  if [[ $OS == "centos" ]]; then
+    PYTHON="python"
+    PIP_PKG="$PYTHON-pip"
     PIP="pip"
     PKG="yum"
     ENABLE_REPO=
-    PKG_EXTRA="yum-utils epel-release git-core desktop-file-utils flatpak ostree skopeo python2-libmodulemd2"
+    PKG_EXTRA="yum-utils git-core desktop-file-utils flatpak ostree skopeo ${PYTHON}2-libmodulemd2"
```

Comment:
IMO `${PYTHON}2` does not make much sense, ideally the `$PYTHON` variable should already include the version number (but that might break something in the centos workflow, in which case leaving it harcoded as `python2-libmodulemd2` would be better IMO)

Reply:
ok